frontend/src/__tests__/cypress/cypress/pages/home/homeAIFlow.ts

```diff
@@ -0,0 +1,41 @@
+import { Contextual } from '~/__tests__/cypress/cypress/pages/components/Contextual';
+
+class AIFlowCard extends Contextual<HTMLElement> {}
+
+export class HomeAIFlow extends Contextual<HTMLElement> {
```

Comment:
Is this really needing to be contextual? If so, then the child queries should be chained with find(), if not just remove the extends.